### PR TITLE
EVG-5403 (feat) PerfBB - links for task and bv

### PR DESCRIPTION
**Pull request by unrealsolver**
perfbb-links

### CODE REVIEW

File: public/static/app/common/EvgUtil.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+mciModule.factory('EvgUtil', function() {
+  var DATE_ID_FORMAT = 'YY_MM_DD_HH_mm_ss'
+
+  // Replaces all '-' with '_'
+  function normalize(text) {
+    return text.replace(/-/g, '_')
+  }
+
+  // internal function which generates evergreen id from given tokens
+  function generateId() {
```
Comment (john-m-liu) [inline public/static/app/common/EvgUtil.js]: Could we make this function explicitly take in an array of arguments which it will join? The current implementation is hard to follow

**Comment by unrealsolver** [comment]
>feels very fragile

If no one will change id generation format on backend, this will work an eternity.
>Do you not have access to their IDs via a query?

There are, obviously, no ids in the response. Potentially, I can make async call(s) to Evergreen, however, perfdiscovery page shows it isn't the best option due to ridiculous number of reqeusts and megabytes.

**Comment by unrealsolver** [comment]
Also, integration tests might help.
